CHANGELOG.rst

```diff
@@ -10,6 +10,7 @@ Yet another overdue... hotfix. Sorry this took so long.
 * The false positive for indented function parameters in namespaces was eradicated. (https://github.com/cpplint/cpplint/pull/304)
 * build/include-what-you-use now recognizes c-style headers, such as <stdio.h> for symbols from <cstdio>. (https://github.com/cpplint/cpplint/pull/319)
 * The warning on non-const references (runtime/references) is now disabled by default pursuant to the May 2020 Google style guide update. (https://github.com/cpplint/cpplint/pull/305)
+* Ruff was ran on the project to improve performance and reader comprehension thanks to @cclauss.
```

Comment:
```suggestion
* Ruff lint was run on the project to use more list, dict, and set comprehensions and to improve performance.  @cclauss.
```

Reply:
1. That would be for a different PR, maybe the bump release date one.
2. There's way more changes to improve performance than comprehensions (there are many readability linter changes), and I don't see the need to single this out.